components/camel-chatscript/src/main/docs/camel-chatscript-component.adoc

```diff
@@ -0,0 +1,75 @@
+[[ChatScript-component]]
+== ChatScript Component
+
+*Available as of Camel version 2.23*
+
+The *ChatScript:* component allows you to interact with ChatScript Server and have converstations. This component is stateless and relies on ChatScript to maintain chat history. 
+
+Original ChatScript is available here : https://github.com/bwilcox-1234/ChatScript
+
+This component expects a JSON having the following fields:
+{
+username:
+botname:
+body:
+}
+
+Refer ChatScriptMessage.java and samples  
+
+Maven users will need to add the following dependency to their `pom.xml`
+for this component:
+
+[source,xml]
+------------------------------------------------------------
+<dependency>
+    <groupId>org.apache.camel</groupId>
+      <artifactId>camel-chatscript</artifactId>
+    <version>x.x.x</version>
+    <!-- use the same version as your Camel core version -->
+</dependency>
+------------------------------------------------------------
+
+### URI format
+[source,java]
+-------------------------------------------------------------------------------------------------------------------------
+ChatScript:host:[port]/BotName[?options]
+-------------------------------------------------------------------------------------------------------------------------
+
+### Options
+
+// component options: START
```

Comment:
It looks like component options have been written by hand. If I remember well, they should be auto-generated from annotations in the component class.

Reply:
Hi, Yes I created the document by copy pasting from another component. I will run a full build to check if the build is created correctly are not.